README.md

```diff
@@ -17,18 +17,17 @@ You can also build a binary yourself following the development instructions belo
 Once FastFEC has been installed, you can run the program by calling `fastfec` in your terminal:
 
 ```
-Usage: fastfec [flags] <id, file, or url> [output directory=output] [override id]
+Usage: fastfec [flags] <id or file> [output directory=output] [override id]
 ```
 
 - `[flags]`: optional flags which must come before other args; see below
-- `<id, file, or url>` is either
-  - a numeric ID, in which case the filing is streamed from the FEC website
+- `<file or id>` is either
   - a file, in which case the filing is read from disk at the specified local path
-  - a url, in which case the filing is streamed from the specified remote URL
+  - a numeric ID (only works with `--print-url`): prints the possible URLs the filing lives on the FEC docquery website
 - `[output directory]` is the folder in which CSV files will be written. By default, it is `output/`.
-- `[override id]` is an ID to use as the filing ID. If not specified, this ID is pulled out of the first parameter as a numeric component that can be found at the end of the path/URL.
+- `[override id]` is an ID to use as the filing ID. If not specified, this ID is pulled out of the first parameter as a numeric component that can be found at the end of the path.
 
-The CLI will download or read from disk the specified filing and then write output CSVs for each form type in the output directory. The paths of the outputted files are:
+The CLI will read the specified filing from disk and then write output CSVs for each form type in the output directory. The paths of the outputted files are:
```

Comment:
Or stdin?

Reply:
I think line 34 a few lines later clarifies "You can also pipe the output of another command in by following this usage"

Welcome changes if you think it would help introducing that concept earlier

Reply:
Covered in the next paragraph.